Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM ubuntu:bionic
+FROM ubuntu:24.04
 MAINTAINER LanCache.Net Team <team@lancache.net>
 ARG DEBIAN_FRONTEND=noninteractive
 RUN \
   apt-get -y update && apt-get -y upgrade && \
-  apt-get -y install python3-pip curl wget bzip2 locales tzdata && \
+  apt-get -y install python3-pip curl wget bzip2 locales tzdata --no-install-recommends && \
   locale-gen en_GB.utf8 && \
-  update-locale LANG=en_GB.utf8
+  update-locale LANG=en_GB.utf8 && \
+  apt-get -y clean && \
+  rm -rf /var/lib/apt/lists/*
 RUN \
-  pip3 install supervisor && \
+  pip3 install supervisor --break-system-packages && \
```

Comment:
Newer versions of Python changed how Python-specific packages are managed between the OS package manager and `pip` in [pep-0668](https://peps.python.org/pep-0668/).

Installing **supervisor** the usual way with `pip3 install supervisor` now throws the following error:
![image](https://github.com/lancachenet/ubuntu/assets/766513/d3c858a2-b33c-41a9-89c6-a8fbeba78c34)

From my understanding it used to be possible for Python packages installed via `pip` to be installed in an implicit global context, which had the potential to run into conflicts with versions of the package that could be installed via the OS package manager.

Trying to install **supervisor** via `apt` with the suggested `apt install python3-supervisor` shows that the package is not included in the apt repository, which to my knowledge means that there isn't a chance of collision.
![image](https://github.com/lancachenet/ubuntu/assets/766513/033b6b5e-7d35-4e46-9f7d-b1568d530765)

`--break-system-packages` just tells `pip` to shut up and install the package the same way that it used to,  I believe that the flag is intended to confirm that you understand the potential issues it can create.  Since this is a single package isolated to just this container, I don't see any issues with keeping it this way.  From my testing it doesn't appear to cause any issues.

Reply:
Thanks for the explanation. This seems like a reasonable change.